src/quat.js

```diff
@@ -447,20 +447,21 @@ export function fromMat3(out, m) {
 }
 
 /**
- * Creates a quaternion from the given euler angle x, y, z.
+ * Creates a quaternion from the given euler angle x, y, z using the provided intrinsic order for the conversion.
  *
  * @param {quat} out the receiving quaternion
  * @param {x} Angle to rotate around X axis in degrees.
  * @param {y} Angle to rotate around Y axis in degrees.
  * @param {z} Angle to rotate around Z axis in degrees.
+ * @param {order} Intrinsic order for conversion, so far only 'xyz' and 'zyx' is supported.
```

Comment:
These docs are still out of date after adding support for the rest of the rotation orders specifically this part:

>  so far only 'xyz' and 'zyx' is supported.

It might also be worth noting that only ["Tait-Bryan angles"](https://en.wikipedia.org/wiki/Euler_angles#Chained_rotations_equivalence) are supported (as opposed to "Proper Euler angles" which specify a rotation about the same axis twice) which I saw the [three.js docs](https://threejs.org/docs/#api/en/math/Euler.order) clarify.

Reply:
The default value for `order` might need to explicitly documented as well? I'm not sure if the docs generation process automatically detects inline parameter defaults.

Reply:
Done.